source/support.tex

```diff
@@ -1346,10 +1346,10 @@
 
 \begin{codeblock}
 namespace std {
-  using int8_t         = @\textit{signed integer type}@;  // optional
-  using int16_t        = @\textit{signed integer type}@;  // optional
-  using int32_t        = @\textit{signed integer type}@;  // optional
-  using int64_t        = @\textit{signed integer type}@;  // optional
+  using int8_t         = @\textit{signed integer type}@;    // optional
+  using int16_t        = @\textit{signed integer type}@;    // optional
+  using int32_t        = @\textit{signed integer type}@;    // optional
+  using int64_t        = @\textit{signed integer type}@;    // optional
 
```

Comment:
Could you please put in the whitespace fixes right away? This makes the "hard" parts easier to spot.

Reply:
Sure, will do.

Reply:
Reverting those changes for now, since they don't take italic correction into account, either. Once we have direction, I'll separate cosmetic and substantive change.